drivers/backup/portworx/portworx.go

```diff
@@ -798,6 +798,7 @@ func (p *portworx) WaitForRunning(
 	var backupErr error
 
 	t := func() (interface{}, bool, error) {
+		logrus.Debugf("WaitForRunning inspect backup state for %s", req.Name)
```

Comment:
is this needed or was it for debugging?

Reply:
Well, I guess it is needed to track backup which backup we are waiting for.